### ISSUE TITLE
[feature] AZERTY keyboard

### ISSUE THREAD

**Issue by Naoya99** [MrNerdy42/Keyboard-Wizard#25]
Hello,
would it be possible to add in the pages the azerty keyboard ?

![image](https://user-images.githubusercontent.com/45720987/80870379-5b278d80-8ca6-11ea-929b-8459efeae275.png)

**Comment by MrNerdy42** [comment]
@TeezeR-22 Yes, this is a feature on my docket for after I graduate college and can get back to modding my favorite game.
